zerver/actions/users.py

```diff
@@ -372,7 +372,7 @@ def send_stream_events_for_role_update(
         send_event_on_commit(user_profile.realm, event, [user_profile.id])
 
 
-@transaction.atomic(durable=True)
+@transaction.atomic
```

Comment:
We should do `savepoint=False` here, right, to avoid using savepoints? I'm pretty sure we decided those were always evil.